### PR TITLE
Use append() instead of directly pushing child nodes

### LINKED ISSUE #71: Don't set Container#nodes directly
You should set children by `append()` or any other inset methods (of course, except `push()`).

Here we have same problem as with `push()`: PostCSS will not fix issues for you as result it is easy to broke AST.

In result we have issue in other PostCSS plugins like this: https://github.com/ben-eb/cssnano/issues/163#issuecomment-208214467

### PR DESCRIPTION
**Pull request by tatjsn**
This resolves #71

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -150,14 +150,17 @@ const processor = postcss.plugin('postcss-modules-scope', function(options) {
     let exportedNames = Object.keys(exports);
     if (exportedNames.length > 0) {
       css.append(postcss.rule({
-        selector: `:export`,
-        nodes: exportedNames.map(exportedName => postcss.decl({
+        selector: `:export`
+      }));
+      let exportRule = css.nodes[css.nodes.length - 1];
```
Comment (just-boris) [inline src/index.js:155]: Why not just:

``` js
const exportRule = postcss.rule({selector: `:export`});
exportedNames.forEach(name => exportRule.append(/*some code*/));
css.append(exportRule);
```

that `css.nodes[css.nodes.length - 1]` reference looks like private-ish api that should not be used

Comment (just-boris) [inline src/index.js:155]: /cc @ai 
am I right?

Comment (ai) [inline src/index.js:155]: Yeap, your version is better. Can you make PR?

Comment (just-boris) [inline src/index.js:155]: created #77

Comment (tatjsn) [inline src/index.js:155]: Sorry for late reply. I just didn't notice that method and @just-boris 's way of doing looks better absolutely.

**Comment by tatjsn** [comment]
Close in favor of #77

**Pull request closed without merging**